easybuild/easyconfigs/b/BamBam/BamBam-1.4-goolf-1.7.20.eb

```diff
@@ -0,0 +1,45 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'ConfigureMake'
+
+name = 'BamBam'
+version = '1.4'
+
+homepage = 'http://udall-lab.byu.edu/Research/Software/BamBam'
+description = "several simple-to-use tools to facilitate NGS analysis"
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://sourceforge.net/projects/bambam/files/']
+sources = ['%(namelower)s-%(version)s.tgz']
+checksums = ['a9b178251d771aafb8c676d30a9af88ea2fc679c2a5672b515f86be0e69238f1']
+
+unpack_options = '--strip-components=1'
+
+dependencies = [
+    ('Perl', '5.22.2'),
+    ('BamTools', '2.4.0'),
+    ('HTSlib', '1.3.1'),
+    ('SAMtools', '1.3.1'),
+]
+
+skipsteps = ['configure', 'install']
+
+buildininstalldir = True
```

Comment:
@pescobar Hold on, isn't `MakeCp` better/cleaner here?

Reply:
no strong preference. As you wish

Reply:
Actually, won't make much of a difference, there's more stuff there than just the binaries...